src/runtime/forester.rs

```diff
@@ -213,9 +213,14 @@ impl Forester {
                                     debug!(target:"trim","attempt to trim is  {:?}", self.trim(&ctx));
                                     ctx.push(child)?;
                                 } else {
-                                    debug!(target:"flow[run]", "tick:{}, {tpe}. The '{child}' is running, decide go up or stay here.",ctx.curr_ts());
+                                    debug!(target:"flow[run]", "tick:{}, {tpe}. The '{child}' is running, decide to go up or stay here.",ctx.curr_ts());
                                     // for parallel node we need to proceed with other children regardless of the current result
-                                    match flow::monitor(tpe, args.clone(), tick_args, &mut ctx)? {
+                                    match flow::monitor(
+                                        tpe,
+                                        init_args.clone(),
```

Comment:
can i suggest returning the formatting?
```
match flow::monitor(tpe, init_args.clone(), tick_args, &mut ctx)? {
```

Reply:
That's just how cargo fmt / rust-analyzer auto-formats it. Does your cargo fmt format it differently?

Reply:
It did not break it into several lines but apparently, I was not using cargo fmt but rather default ide (rustrover) formater. Yeah, keep it as is then